kroxylicious/src/main/resources/log4j2.properties

```diff
@@ -11,11 +11,23 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} <%t> %-5p %c:%L - %m%n
 
-rootLogger.level = WARN
+rootLogger.level = ${env:KROXYLICIOUS_ROOT_LOG_LEVEL:-WARN}
 rootLogger.appenderRefs = stdout
 rootLogger.appenderRef.console.ref = STDOUT
 rootLogger.additivity = false
 
-#logger.netty.name = io.netty
-#logger.netty.level = TRACE
-#logger.netty.additivity = false
+# The DownstreamNetworkLogger and DownstreamFrameLogger loggers are required to allow
+# the virtual cluster logNetwork/logFrames feature to work.  The log level needs to be in sync
+# with the logger level used in the code.
+logger.DownstreamNetworkLogger.name = io.kroxylicious.proxy.internal.DownstreamNetworkLogger
```

Comment:
Should we also be configuring a separate appender for network and or frame logging? So we can default it to file?

Reply:
Yeah, I was wondering about the K8s log everything to stdout model, but I suspect the majority of the time we want frame and network logging is in dev/local envs and keeping them in a separate file feels tempting to me for that usage.